alerts/proxy_drop_ip.py

```diff
@@ -28,6 +30,9 @@ def main(self):
 
         search_query.add_must([QueryStringMatch("details.host: {}".format(ip_regex))])
 
+        for ip in self.config.ip_whitelist.split(","):
```

Comment:
Can we add an example test that verifes we won't alert if the whitelist matches?

Reply:
Added a test for this.